tests/test_conf.ml

```diff
@@ -4,35 +4,43 @@
  *)
 open Alcotest
 open Core
+open Core.Monad
 
 let expected_res =[
-  ("conf_content1.test", Some(List.sort compare ["ex.*\\.json";"tests/"]), true);
-  ("conf_content2.test", Some(List.sort compare ["\\./src/flash"; "tests/coc.*\\.ml";"tests/banana"]), false);
-  ("conf_content3.test", None, true)
+  ("conf_content1.test", List.sort compare ["ex.*\\.json";"tests/"], true);
+  ("conf_content2.test", List.sort compare ["\\./src/flash"; "tests/coc.*\\.ml";"tests/banana"], false);
+  ("conf_content3.test", [], true)
 ]
 
-let apply_option f = function
-  | None -> None
-  | Some el -> Some (f el)
 
 let test_regex_file_eq _ = 
   let aux (a,b,_) =
-    let js = Conf.init_json ("res/" ^ a) in
-    (check (option (list string))) ("regex_content : "^ a) b (apply_option (List.sort compare) (Conf.get_file_regex js))
+    let js = match Conf.init_json ("res/" ^ a) with
+      | Ok elt -> elt
+      | _ -> raise Test_error 
+    in 
+    let res = match Conf.get_file_regex js with
+      | Ok content -> (List.sort compare content)
+      | Error elt -> []
+    in
+    (check (bool)) ("regex_content : "^ a) true (b=res)
```

Comment:
replace `(check (bool)) ("regex_content : "^ a) true (b=res)` by `(check (list string)) ("regex_content : "^ a) b res` : compare string list with the right comparator.

Reply:
It's your test. I don't change anything from the previous test.